package.json

```diff
@@ -20,7 +20,6 @@
   },
   "license": "Apache-2.0",
   "author": "Henrique Doro <opensource@hdoro.dev>",
-  "type": "commonjs",
```

Comment:
Removing `type` fixes error: `module parse failed: 'import' and 'export' may appear only with 'sourcetype: module'` when testing the plugin in a studio client.